lib/langchain/tool/database.rb

```diff
@@ -14,6 +14,13 @@ class Database < Base
       The input to this tool should be valid SQL.
     DESC
 
+    SCHEMA_PARAMETERS = {
+      input: {
+        type: "string",
+        description: "SQL expression"
+      }
```

Comment:
I think this change will be a bit more informative to the LLM:
```suggestion
      sql_query: {
        type: "string",
        description: "SQL query"
      }
```
but then need to rename the argument on the `execute` method too. Will that rename break original tools implementation? maybe can be an enhancement for other pr

Reply:
maybe a change like this could break non-function calling flows. I mean, using tools without function calling